modules/acknowledgements/php/acknowledgements.class.inc

```diff
@@ -28,7 +28,11 @@ namespace LORIS\acknowledgements;
 
 class Acknowledgements extends \NDB_Menu_Filter_Form
 {
-    public $skipTemplate = true;
+    public const PERMISSIONS = array(
```

Comment:
This class should not be changed if you're just adding tests. This const is useless.